lib/main_common.pm

```diff
@@ -1055,6 +1055,7 @@ sub load_console_server_tests {
         loadtest "console/yast2_nfs_server";
     }
     loadtest "console/rsync";
+    loadtest "console/rust";
```

Comment:
IMHO wrong location. Should be in "extra tests" only. https://github.com/os-autoinst/os-autoinst-distri-opensuse/pull/13756/files#diff-051cb1f26a96a4f724250d9182535d868c691e20963ef91a1a7ca109e9538379R1609 and below